helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -378,6 +405,7 @@ public Builder(final String hash, final Parameters parameters) {
       public Resources resources;
       public Map<String, PortMapping> ports;
       public Map<ServiceEndpoint, ServicePorts> registration;
+      public Boolean disableautoRegistration;
```

Comment:
Heh, apparently we did not catch everything while renaming, will update in a bit.